certificate-service/domain/src/main/java/se/inera/intyg/certificateservice/domain/message/service/SetMessagesToHandleDomainService.java

```diff
@@ -12,6 +13,10 @@ public class SetMessagesToHandleDomainService {
 
   public void handle(List<Message> messages) {
     messages.forEach(Message::handle);
+    messages.stream()
+        .filter(message -> message.answer() != null
+            && message.answer().status() == MessageStatus.DRAFT)
```

Comment:
Use isDraft function if we seem fit to implement it!

Reply:
done